### PR TITLE
add flag to remove socket file at the end

### PR DESCRIPTION
**Pull request by bAndie91**

### CODE REVIEW

File: keept.c [hunk 1]
```diff
@@ -136,6 +136,7 @@ static void usage(const char * prgname, int more)
 #if HAVE_ABSTRACT_SOCKET_NAMESPACE
 		"     @: use socket in abstract namespace" nl
 #endif
+		"     S: remove socket at the end" nl
```
Comment (domo141) [inline keept.c]: tried to use lowercase flags only (although @ may require shift/alrgr etc, @ was just so good... ;D),
anyway perhaps 'u' -- like unlink instead ?

File: keept.c [hunk 2]
```diff
@@ -1071,7 +1074,9 @@ int main(int argc, const char * argv[])
 	die("Connecting to socket failed:");
     }
     G.redraw_mode = G.redraw_mode | 128; // bit 7 for immediate first connection
-    return attached(s);
+    int aret = attached(s);
+    if(remove_socket) unlink(sockname);  // ignore any error (socket file may has been moved)
```
Comment (domo141) [inline keept.c]: space after if -- if (remove_socket)

also, I think it should say '(socket file may have been moved)'

File: keept.c (hunk 1 above)
Comment (bAndie91) [reply keept.c]: i'm totally good with 'u' flag. btw 'S' was meant to be mnemonic for "socket" and uppercase usually means negation among cli options. '@' is really a good mnemonic.

**Comment by domo141** [comment]
Andreas,

amended your commit a bit -- pushed it to "branch"/ref topic-03a13e2
( see https://github.com/domo141/keept/commit/03a13e208c57c6de6cb5407b992784476c36c934 )

If that looks good to you (you marked as Author) I'll push that to "master".

**Comment by bAndie91** [comment]
LGTM

**Comment by domo141** [comment]
merged from command line (git push origin 03a13e2:master)

**Pull request closed without merging**